### PR TITLE
BAPL-1173 Designer - Add support for generic Service Task

### PR DESCRIPTION
**Pull request by treblereel**
https://issues.jboss.org/browse/BAPL-1173
ps: tests will be added later

### CODE REVIEW

**Comment by romartin** [comment]
Hey @treblereel 
Thanks for this PR! Adding @LuboTerifaj as reviewer too.

BTW the build failed due to some failed tests:
```
org.kie.workbench.common.stunner.bpmn.backend.converters.tostunner.tasks.BaseTaskConverterTest.convertBusinessRuleServiceTask
org.kie.workbench.common.stunner.bpmn.backend.converters.tostunner.tasks.TaskConverterTest.convertBusinessRuleServiceTask
org.kie.workbench.common.stunner.bpmn.backend.service.diagram.marshalling.tasks.ServiceTaskTest.testBasicUnmarshall
org.kie.workbench.common.stunner.bpmn.backend.service.diagram.marshalling.tasks.ServiceTaskTest.testBasicBidi
org.kie.workbench.common.stunner.bpmn.backend.service.diagram.marshalling.tasks.ServiceTaskTest.testBasicMarshall
```

File: kie-wb-common-stunner/kie-wb-common-stunner-sets/kie-wb-common-stunner-bpmn/kie-wb-common-stunner-bpmn-client/src/main/resources/org/kie/workbench/common/stunner/bpmn/client/resources/images/shapes/bpmn-shapes.css [hunk 1]
```diff
@@ -116,6 +116,14 @@
     opacity: 0;
 }
 
+.task_group_generic_service {
+    opacity: 0;
+}
+
+.task_generic_service_icon {
+    opacity: 0;
```
Comment (romartin) [inline kie-wb-common-stunner/kie-wb-common-stunner-sets/kie-wb-common-stunner-bpmn/kie-wb-common-stunner-bpmn-client/src/main/resources/org/kie/workbench/common/stunner/bpmn/client/resources/images/shapes/bpmn-shapes.css]: that's wrong - setting the opactity for the icon to 0 will make it not visible :)
change it to  `fill: #393f44;`

**Comment by romartin** [comment]
Please @treblereel 

It's important that you can fix the actual build, so @LuboTerifaj  or @hasys can test it. Actually it's failing due to some failed tests:

```
org.kie.workbench.common.stunner.bpmn.backend.converters.tostunner.tasks.BaseTaskConverterTest.convertBusinessRuleServiceTask
org.kie.workbench.common.stunner.bpmn.backend.converters.tostunner.tasks.TaskConverterTest.convertBusinessRuleServiceTask
org.kie.workbench.common.stunner.bpmn.backend.service.diagram.marshalling.tasks.ServiceTaskTest.testBasicUnmarshall
org.kie.workbench.common.stunner.bpmn.backend.service.diagram.marshalling.tasks.ServiceTaskTest.testBasicBidi
org.kie.workbench.common.stunner.bpmn.backend.service.diagram.marshalling.tasks.ServiceTaskTest.testBasicMarshall
```

Please guys, it's IMPORTANT to complete this today (Thu) and trying to merge it! Give it priority plz! 

Thanks!

**Comment by treblereel** [comment]
retest this please

**Comment by LuboTerifaj** [comment]
Hi @romartin , just FYI, since the PR is created by @treblereel , the review should be directed to @hasys . For more information see our mailing list. Thanks!

**Comment by romartin** [comment]
reassigned review to @hasys

**Comment by hasys** [comment]
Jenkins execute full downstream build

**Pull request merged**